ci-operator/config/netobserv/network-observability-console-plugin/netobserv-network-observability-console-plugin-main.yaml

```
@@ -0,0 +1,133 @@
+base_images:
+  go-builder-base:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "9"
+  web-builder-base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  use_build_cache: true
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  inputs:
+    go-builder-base:
+      as:
+      - docker.io/library/golang:1.21
+    web-builder-base:
+      as:
+      - docker.io/library/node:16-alpine
+  to: network-observability-console-plugin
+- build_args:
+  - name: BUILDSCRIPT
+    value: :standalone
+  dockerfile_literal: |
+    ARG BUILDSCRIPT=
+    ARG LDFLAGS=
+    ARG TARGETPLATFORM=linux/amd64
+    ARG BUILDPLATFORM=linux/amd64
+    FROM --platform=$BUILDPLATFORM docker.io/library/node:16-alpine as web-builder
+    USER node
+
+    ARG BUILDSCRIPT
+    ARG TARGETPLATFORM
+    ARG TARGETARCH=amd64
+    WORKDIR /opt/app-root
+
+    COPY --chown=node web/package.json web/package.json
+    COPY --chown=node web/package-lock.json web/package-lock.json
+    WORKDIR /opt/app-root/web
+    RUN npm ci
+
+    WORKDIR /opt/app-root
+    COPY --chown=node web web
+    COPY mocks mocks
+
+    WORKDIR /opt/app-root/web
+    RUN npm run format-all
+    RUN npm run build$BUILDSCRIPT
+
+    FROM --platform=$BUILDPLATFORM docker.io/library/golang:1.21 as go-builder
```

```suggestion
    FROM docker.io/library/golang:1.21 as go-builder
```

you'd also avoid using the image from docker.io. We have images in app.ci that can be used: e.g., `registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-builder-multi-openshift-4.16`

---

yes, I will be replacing with the redhat images. Here we're trying to get cypress tests to work before finalizing.